aqt/main.py

```diff
@@ -1159,11 +1159,11 @@ def deleteUnused(self, unused, diag):
                 now = time.time()
                 if now - lastProgress >= 0.3:
                     lastProgress = now
```

Comment:
The original string had the same problem, but this needs to be updated to use ngettext("single form %d", "plural form %d", count) % count

Reply:
Sorry, that comment was intended for the '%d files remaining' line.